parser/src/parse/generated.rs

```diff
@@ -4,4 +4,4 @@
 #![allow(unused_attributes)]
```

Comment:
This file changed when running the tests locally before making any changes. I'm not sure if that's expected or if there's something I need to do differently when running the tests.